src/Components/Server/src/Circuits/RemoteRenderer.cs

```diff
@@ -279,10 +279,10 @@ static Log()
                     EventIds.UnhandledExceptionRenderingComponent,
                     "Unhandled exception rendering component: {Message}");
 
-                _beginUpdateDisplayAsync = LoggerMessage.Define<string>(
+                _beginUpdateDisplayAsync = LoggerMessage.Define<long, int, string>(
                     LogLevel.Trace,
                     EventIds.BeginUpdateDisplayAsync,
-                    "Begin remote rendering of components on client {ConnectionId}.");
+                    "Sending render batch {batchId} of size {dataLength} bytes to client {ConnectionId}.");
```

Comment:
nit: Use PascalCase for structured log keys

Reply:
Done, thanks.